dev-data/about-video.data.ts

```diff
@@ -0,0 +1,4 @@
+export const video = {
```

Comment:
I suggest `videoTitleLocalized`

Reply:
I was just about to write a comment about this variable name :)

Reply:
Done